src/services/OpenAPIParser.ts

```diff
@@ -324,9 +329,25 @@ export class OpenAPIParser {
         receiver.required = (receiver.required || []).concat(required);
       }
 
+      if (oneOf !== undefined) {
+        receiver.oneOf = [...(receiver?.oneOf || []), ...oneOf];
```

Comment:
This is not 100% valid. We can't just merge oneOf's. But this is not needed. We hoist oneOfs to the top-level before.